mood-analysis.rb

```diff
@@ -28,5 +36,49 @@ def analyze_mood(words)
   "05/11 Yay, yay, yay! I'm having a awfuly great day."
 ]
 
-puts analyze_mood(text[0])
-puts analyze_mood(text[1])
+def moods(text)
+  text.each do |blog|
+    face = analyze_mood(blog)
+    puts blog[0..5] + face
+  end
+end
+
+def happy_days(text)
+  count = 0
+  days = 0
+  text.each do |blog|
+    face = analyze_mood(blog)
+    days += 1
+    count += 1 if face == ":-)"
+    break if count == 3
```

Comment:
There's no reason to put a break here instead of just putting the return right here as soon as you meet the condition you are looking for.